### PR TITLE
docs: fix playground

### PR DESCRIPTION
**Pull request by ludofischer**
Let people choose the default and avanced preset in the playground.

Start solving #1162

### CODE REVIEW

File: site/src/pages/playground.js [hunk 1]
```diff
@@ -1,21 +1,15 @@
 import React, { useState } from 'react';
 import Layout from '@theme/Layout';
+import Editor from '@monaco-editor/react';
```
Comment (ludofischer) [inline site/src/pages/playground.js:3]: Not really happy about this editor. It loads from a CDN, seems a bit hard to understand how to get it to run from our own instance

File: site/docusaurus-webpack-plugin.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      return {
+        resolve: {
+          fallback: {
+            path: require.resolve('path-browserify'),
+            os: require.resolve('os-browserify/browser'),
+            fs: false,
+          },
+        },
+        plugins: [
+          new webpack.EnvironmentPlugin({ BROWSERSLIST_DISABLE_CACHE: true }),
```
Comment (ludofischer) [inline site/docusaurus-webpack-plugin.js:16]: Without this browserslist crashes in the browser.

File: site/docusaurus-webpack-plugin.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+const webpack = require('webpack');
+
+module.exports = function () {
+  return {
+    name: 'webpack-cssnano-docusaurus-plugin',
+    configureWebpack() {
+      return {
+        resolve: {
+          fallback: {
+            path: require.resolve('path-browserify'),
```
Comment (ludofischer) [inline site/docusaurus-webpack-plugin.js:10]: Required by `normalize-url` and `svgo`.

**Review by alexander-akait — APPROVED** [review]
Looks good

**Comment by ludofischer** [comment]
I'm going to merge it as at least people can test the default preset now. It needs more work because the user can type anything in the `config` editor and in many cases the playground cannot parse the config.

**Pull request merged**